lib/ui/page/home/page/chat/info/view.dart

```diff
@@ -517,18 +541,25 @@ class ChatInfoView extends StatelessWidget {
           onPressed: () => router.dialog(c.chat!.chat.value, c.me),
           child: const SvgIcon(SvgIcons.chat),
         ),
-        const SizedBox(width: 28),
-        AnimatedButton(
-          onPressed: () => c.call(true),
-          child: const SvgIcon(SvgIcons.chatVideoCall),
-        ),
-        const SizedBox(width: 28),
-        AnimatedButton(
-          key: const Key('AudioCall'),
-          onPressed: () => c.call(false),
-          child: const SvgIcon(SvgIcons.chatAudioCall),
-        ),
-        const SizedBox(width: 20),
+        const SizedBox(width: 8),
+        c.isMonolog == false
+            ? Row(
+                children: [
+                  const SizedBox(width: 20),
+                  AnimatedButton(
+                    onPressed: () => c.call(true),
+                    child: const SvgIcon(SvgIcons.chatVideoCall),
+                  ),
+                  const SizedBox(width: 28),
+                  AnimatedButton(
+                    key: const Key('AudioCall'),
+                    onPressed: () => c.call(false),
+                    child: const SvgIcon(SvgIcons.chatAudioCall),
+                  ),
+                  const SizedBox(width: 20),
+                ],
+              )
+            : SizedBox(),
```

Comment:
Why build `SizedBox` at all, if you can just not being anything when `!c.isMonolog`?